### ISSUE TITLE
 ResizeObserverOptions is not an array

### ISSUE THREAD

**Issue by prideout** [mdn/content#18818]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserverEntry/devicePixelContentBoxSize

### What specific section or headline is this issue about?

ResizeObserverEntry

### What information was incorrect, unhelpful, or incomplete?

The example shown on this page is incorrect and inconsistent with the spec and other examples at MDN, because the box options should not be an array:

    observer.observe(canvas, {box: ['device-pixel-content-box']});

### What did you expect to see?

It should be:

    observer.observe(canvas, {box: 'device-pixel-content-box'});

### Do you have any supporting links, references, or citations?

It is not an array in the spec, nor in this other MDN example:

https://www.w3.org/TR/resize-observer/
https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver/observe

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/resizeobserverentry/devicepixelcontentboxsize`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserverEntry/devicePixelContentBoxSize
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/resizeobserverentry/devicepixelcontentboxsize/index.md
* Last commit: https://github.com/mdn/content/commit/875eef5ce2cd92c56d8ab8cb58c85d85bb341d4a
* Document last modified: 2022-06-02T05:03:51.000Z

</details>

**Comment by teoli2003** [comment]
The spec says it is a `FrozenArray` so it looks correct to me.

**Comment by prideout** [comment]
Link?  I don't see the box options represented as a `FrozenArray` in either the November or February drafts:
- https://drafts.csswg.org/resize-observer/#resize-observer-interface
- https://www.w3.org/TR/resize-observer/#resize-observer-interface

(However some of the `ResizeObserverEntry` fields are indeed arrays.)

Here are the relevant snippets from the first of the above links:

```webidl
enum ResizeObserverBoxOptions {
    "border-box", "content-box", "device-pixel-content-box"
};
```

```webidl
[Exposed=(Window)]
interface ResizeObserver {
    constructor(ResizeObserverCallback callback);
    undefined observe(Element target, optional ResizeObserverOptions options = {});
    undefined unobserve(Element target);
    undefined disconnect();
};
```

```js
// Observe the content-box
ro.observe(document.querySelector('#menu'), { box: 'content-box' });

// Observe just the border box. Replaces previous observation.
ro1.observe(document.querySelector('#menu'), { box: 'border-box' });
```

**Comment by teoli2003** [comment]
I read the wrong entry. You are right. 

Fix incoming in #18821.
